algorithms/frontier_based_exploration.py

```diff
@@ -111,8 +114,46 @@ def visualize(occupancy_map, state, scan_angles, scan_ranges, footprint, path, r
     cv2.waitKey(wait_key)
 
 
+def visualize_in_gym(gym_env, exp_map, path, pose):
+    """
+    Alternative visualization method using gym rendering function
+    :param gym_env RandomAisleTurnEnv: object storing the robot and environment state in bc_gym_planning_env standard
+    :param exp_map Costmap: occupancy map in bc_exploration standard
+    :param path array(N, 3)[float]: containing an array of poses for the robot to follow
+    :param pose array(3)[float]: corresponds to the pose of the robot [x, y, theta]
+    """
+
+    gym_compat_map = np.zeros(exp_map.data.shape,dtype=np.uint8)
+    gym_compat_map[exp_map.data == Costmap.OCCUPIED] = CostMap2D.LETHAL_OBSTACLE
+    gym_compat_map[exp_map.data == Costmap.FREE] = CostMap2D.FREE_SPACE
+    gym_compat_map[exp_map.data == Costmap.UNEXPLORED] = CostMap2D.NO_INFORMATION
+
+    gym_costmap = CostMap2D(np.flipud(gym_compat_map),exp_map.resolution,exp_map.origin.copy())
+
```

Comment:
you can use Costmap.from_brain_costmap(), it does the same thing. But it's good that you implemented this anyway so you know the difference between the costmaps.